Modules/Write-ObjectToSQL/Write-ObjectToSQL.psm1

```diff
@@ -337,7 +337,18 @@ function Write-ObjectToSQL
                    ParameterSetName='mssql',
                    ValueFromPipeline=$False)]
         [ValidateNotNullorEmpty()]
-        [string]$PrimaryKey
+
+        [string]$PrimaryKey,
+
+        # Optional Switch to Drop Table before Data is loaded.
+        # This Switch is Usefull if you want to Drop your Table before. (No data Append)
+        [Parameter(Mandatory=$False,
+                   ParameterSetName='mssql',
+                   ValueFromPipeline=$False)]
+        [Parameter(Mandatory=$False,
+                   ParameterSetName='othersql',
+                   ValueFromPipeline=$False)]
+        [switch]$DropTable
```

Comment:
I think there may be a conflict with the `DoNotCreateTable` switch ..
maybe use a `[bool]` here instead of a `[switch]` and warn if they confilict?
```
(DropTable -And DoNotCreateTable) 
```
problably doesn't make much sense ...